src/libraries/Microsoft.Extensions.Caching.Abstractions/README.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft.Extensions.Caching
+
+Caching is combined with a core caching abstraction under `Microsoft.Extensions.Caching.Abstractions` that allows for implementing general purpose memory and distributed caches, with integration for Redis and SqlServer.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/core/extensions/caching.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
```

Comment:
could you please add `, bug fixes, ` to here. I know you got the text from the central file. I asked Jose if we can change the centralized doc